gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/handler/common/RedirectUriValidationHandler.java

```diff
@@ -32,9 +40,9 @@
  * The authorization server validates the request to ensure that all parameters are valid.
  * If the request is valid, the authorization server authenticates the resource owner and obtains
  * an authorization decision (by asking the resource owner or by establishing approval via other means).
- *
+ * <p>
```

Comment:
```suggestion
 *
```

Reply:
We *do* publish the javadocs, so I don't think it makes sense to remove automatic formatting fixes.

Before:
![image](https://github.com/gravitee-io/gravitee-access-management/assets/165262917/4475abc9-9f69-4e79-8aed-734b9595ce9a)

After:
![image](https://github.com/gravitee-io/gravitee-access-management/assets/165262917/6058985c-53af-4c69-a700-381da6d10ed2)